NEW.md

```diff
@@ -0,0 +1,9 @@
+# sidora 1.0.0 - Mnemosyne
```

Comment:
If we call it NEWS.md we don't have to ignore it in the .Rbuildignore, I think. 

e.g. https://github.com/poseidon-framework/poseidonR/blob/master/NEWS.md

Reply:
![image](https://user-images.githubusercontent.com/17950287/135414885-5b91152c-f889-4e15-912a-66c8b73012f0.png)
It's not in there already?

Reply:
Exactly. We currently get this NOTE in `devtools::check()`:

```
N  checking top-level files
   Non-standard file/directory found at top level:
     ‘NEW.md’
```

To make this disappear we can either add `NEW.md` to the buildignore file OR rename the file to NEWS.md (afaik). The latter is more standard.

Reply:
Oh sorry completely misread . didn't realise the typo in NEWS.md :laugh: